xunit.runner.visualstudio.testadapter/VsTestRunner.cs

```diff
@@ -307,9 +309,13 @@ static bool IsXunitTestAssembly(string assemblyFileName)
             if (platformAssemblies.Contains(Path.GetFileName(assemblyFileName)))
                 return false;
 
+#if DOTNET_CORE
+            return true;
```

Comment:
I've added a method to verify this using Microsoft.Extensions.DependencyModel package.